### ISSUE TITLE
Right click - Filter Target Path

### ISSUE THREAD

**Issue by theevilbit** [redcanaryco/mac-monitor#6]
**At a high level -- can you summarize your request?**
Would be nice to be able to quickly filter events based on Target Paths, just like we have an option for Process Path filtering.

**What is the current alternative solution?**
Use muted paths in main settings, or export events and filter them manually.

**Are there "In-the-Wild" threats or corresponding ATT&CK techniques that exist for which this telemetry would be helpful?**
No, this only helps with general event filtering.

**Anything else?**
No

**Comment by Brandon7CC** [comment]
This is one I'm happy the community brought in! Target path filtering is an additional option I think would be a relatively easy ask 😄

**Comment by AndrewMohawk** [comment]
Also would be nice to have filtering by the other columns available as well (Source process, Source Signing ID)

**Comment by Brandon7CC** [comment]
Implemented! See `v1.0.4` -- thank you Csaba!
